docs/components/inputMapper.base.md

```diff
@@ -0,0 +1,300 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# `gamepad.inputMapper.base`
+
+<!-- TODO: Add links to the inputMapper component's documentation -->
+
+This component transforms the gamepad inputs into actions. It is derived from the combination of two grades - the
+[`navigator`](navigator.md) grade, which reads and stores the gamepad inputs in the form of model data, and the
+[`configMaps`](configMaps.md) grade provides a configuration map for the gamepad inputs. The navigation features
+available with this component are limited to the **intra-web page navigation** features. (Use the `inputMapper`
+component for **inter-web page navigation** features)
+
+## Using this grade
+
+For the actions to work, you need to use the `gamepad.inputMapperUtils.content` namespace unless the `gamepad` namespace
```

Comment:
I'm not sure I get what you're going for here.  Can you describe in a comment what you were trying to say?

Reply:
The invoker functions for navigation use the namespace `gamepad.inputMapperUtils.content`. So, I added that they need to use this namespace for navigation in cases where they don't register the `gamepad` namespace.

Reply:
I guess I assumed that including your work was enough to define that as far as `fluid.invokeGlobal` needs (that backs the `funcName` stuff IIRC).  But I guess you have tried this with a different namespace and had problems?

Reply:
> But I guess you have tried this with a different namespace and had problems?

@the-t-in-rtf I haven't tried this. But I assumed that Infusion doesn't provide access to sibling namespaces unless the parent namespace is specified. That's the reason I added it.

Reply:
It's not needed.  Including the file that defines the required namespaces is sufficient.  To give a node example, say I have a file called `spaceone.js` that defines a namespace and a static function:

```
var fluid = require("infusion");

var spaceone = fluid.registerNamespace("spaceone");

spaceone.sayHello = function () {
    console.log("Hello.");
}
```

If I require that file from another file, I can make use of the first file's functions without ever defining that namespace:

```
var fluid = require("infusion");

require("./spaceone.js");

var spacetwo = fluid.registerNamespace("spacetwo");
fluid.defaults("spacetwo.grade", {
    gradeNames: ["fluid.component"],
    listeners: {
        "onCreate.sayHello": {
            funcName: "spaceone.sayHello"
        }
    }
});

spacetwo.grade();
```

The alternate namespace is visible from the second file and the code works as expected.

Reply:
@the-t-in-rtf thanks for the example! It does clarify a few things that I misunderstood. Also, I've updated the pull. Kindly take a look over it.